rmgpy/pdep/collision.pyx

```diff
@@ -101,8 +101,6 @@ cdef class SingleExponentialDown:
             except quantity.QuantityError:
                 self._alpha0 = quantity.Energy(value)
 
-            self._alpha0 = quantity.Energy(value)
```

Comment:
This extra line was left over from some improper merging and is a bug on it's own write. If there is no error in the above try statement, it will overwrite the proper method of setting `_alpha0`.

It's in a commit about errors since it improperly is handling the QuantityError.